Common/Constants/include/CommonConstants/PhysicsConstants.h

```diff
@@ -63,6 +63,8 @@ enum Pdg {
   kPhi = 333,
   kSigmaC0 = 4112,
   kSigmaCPlusPlus = 4222,
+  kSigmaCStar0 = 4114,
+  kSigmaCStarPlusPlus = 4224,
```

Comment:
Did you forget to `git add` the Python script?

Reply:
Indeed I forgot. Going to fix it soon.

Reply:
@vkucera done